builder.config.json

```diff
@@ -39,15 +39,15 @@
           "name": "home",
           "path": "\/",
           "methods": ["GET"]
-        }
-      },
-      "trees": {
+        },
         "editTree": {
           "name": "editTree",
-          "path": "\/trees\/{treeId}\/{component?}\/{mode?}",
-          "params": ["treeId", "component", "mode"],
+          "path": "\/flows\/{flowId}\/{component?}\/{mode?}",
+          "params": ["flowId", "component", "mode"],
           "methods": ["GET", "HEAD"]
-        },
+        }
+      },
+      "trees": {
```

Comment:
@poojahpatil90 , Wondering why we moved out only `editTree` to `"flows"` not the whole thing under `"trees"` ?

Reply:
@safydy-r  Good question!  Currently we have `editTree` route being actively used. I believe the other routes like `resumeableAudioUpload` etc have not yet been used in flow builder. I am concerned that moving the other routes might break something in the future when we start using them. It would be better if we move it when we use the other routes as we can test that it does not break things. 
Let me know your thoughts on this?

Reply:
Thanks for clarifying.
I'm still hesitating because we cannot leave a TODO here to remind us in the future.
But, let's leave it as it is for now then.